daemon/integration/integration_test.go

```diff
@@ -0,0 +1,66 @@
+package integration_test
+
+import (
+	"fmt"
+	"github.com/scootdev/scoot/daemon/client/cli"
+	"github.com/scootdev/scoot/daemon/client/conn"
+	"github.com/scootdev/scoot/daemon/integration"
+	"github.com/scootdev/scoot/daemon/server"
+	"io/ioutil"
+	"os"
+	"path"
+	"testing"
+)
+
+func TestIntegration(t *testing.T) {
+	tempDir, err := ioutil.TempDir("", "scoot-listen-")
+	if err != nil {
+		t.Fatal("could not make temp directory", err)
+	}
+	// defer os.RemoveAll(tempDir)
+
+	scootDir := path.Join(tempDir, "scoot")
+	err = os.Setenv("SCOOTDIR", scootDir)
+
+	s, err := server.NewServer(nil)
+	if err != nil {
+		t.Fatal("could not make server", err)
+	}
+
+	l, err := server.Listen()
+
+	go func() {
+		s.Serve(l)
+	}()
+
+	if err != nil {
+		t.Fatal("could not set env", err)
+	}
+
+	dialer, err := conn.UnixDialer()
+	if err != nil {
+		t.Fatal("could find Scoot Daemon", err)
+	}
+
+	err = testEcho(dialer)
+}
+
+func testEcho(dialer conn.Dialer) error {
```

Comment:
nit: maybe a different name then testEcho. suggestion: validateEcho  Kind of confusing to have a method named test that isn't actually a test case in a test file.